tools/docker/tests/test_flows.sh

```diff
@@ -664,5 +681,8 @@ main() {
 VERBOSE=false
 SKIP_INIT=false
 STOP_ON_FAILURE=false
+DUMP=false
+DUMPFILE=-
```

Comment:
Is this a typo? Why DUMPFILE=*-*?

Reply:
This issue was addressed in a followup commit; you're not looking at the latest version

`-` is generally interpreted as standard output, which I agree was a mistake to put as a default.